dirs/dirs.go

```diff
@@ -113,6 +113,9 @@ const (
 	// are in the snap confinement environment.
 	CoreLibExecDir   = "/usr/lib/snapd"
 	CoreSnapMountDir = "/snap"
+
+	// Directory with snap data inside user's home
+	UserHomeSnapDir = "snap"
```

Comment:
This will be interesting because we have a bug to allow to move this to `.snap`

Reply:
Nitpick: can you make this a function please. It will be easier to fix later on.

Reply:
OTOH, I'd prefer to leave it as a `const` for now and defer any changes until we have a plan for switching the path globally.